tests/bgp/test_4-byte_asn_community.py

```diff
@@ -0,0 +1,231 @@
+'''
+
+This script is to Verify applied communities manipulate traffic as
+expected between 4-byte and 2-byte AS neighbors.
+
+Step 1: Configure DUT and neighbor with 4Byte ASN
+Step 2: Verify 4-byte BGP session between DUT and neighbor is established
+
+'''
+import logging
+
+import pytest
+import time
+import textfsm
+from tests.common.config_reload import config_reload
+
+logger = logging.getLogger(__name__)
+dut_4byte_asn = 400003
+neighbor_4byte_asn = 400001
+bgp_sleep = 60
+bgp_id_textfsm = "./bgp/templates/bgp_id.template"
+
+pytestmark = [
+    pytest.mark.topology('t2')
```

Comment:
Do you intent to support this only for T2 and not for T0/T1 DUTs?

Reply:
@gechiang Yes, my group has been specifically tasked with creating test cases for T2.